__tests__/app.test.js

```diff
@@ -117,17 +117,7 @@ describe("app", () => {
 
 
   describe("/api/articles/:article_id/comments", () => {
```

Comment:
Think about what exactly this describe block contains  - these are all GET requests